frontend/src/pages/buybacks/BBDetail.tsx

```diff
@@ -342,10 +362,14 @@ export default function BBDetail() {
       onClickEdit={() => setIsModifiable(!isModifiable)}
       onClickCancel={() => {
         setIsModifiable(!isModifiable);
-        window.location.reload();
+        setDate(originalData.date);
```

Comment:
Do we update this when the user clicks submit? I'm worried about the case when the user does edit -> changes something -> submit (and stay on page) -> edit -> cancel. Would it revert back to the wrong data? Or do we update originalData when the user clicks submit?

Reply:
yup it updates. on the CallModifyAPI method when the API request is success on return I update the originalData item to the newest ones so the next edit cancel should work as intended. I have also tried it myself. Let me know if you find something a miss in testing.